packages/scripts/index.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+/* eslint-disable global-require,no-console */
+
+const { getEnv } = require('./utils/env');
```

Comment:
utils folder! shame! ping @glorieux

Reply:
If I rename it to `common` will you be happy ?

Reply:
No I don't care, just kidding :)